frontend/src/components/Dashboard/reviewModal.js

```diff
@@ -1,24 +1,36 @@
 import React, { useState } from "react";
 import { Modal, Button, Form, Select } from "react-bootstrap";
 
-function ReviewModal(props) {
-  const [comment, setComment] = useState();
-  const [rating, setRating] = useState(1);
+import { Rating } from "react-simple-star-rating";
+import CloseButton from "react-bootstrap/CloseButton";
 
-  function ratingChangeHandler(e) {
-    setRating(e.target.value);
-  }
+function ReviewModal(props) {
+  const [comment, setComment] = useState(props.comment);
+  const [rating, setRating] = useState(props.rating);
+  const ratingChangeHandler = (rate) => {
+    if (rate < 1) {
+      rate = 1;
+    }
+    setRating(rate * 2); //rating is 1 - 10 on backend and 0.5 - 5 on frontend so i multiply it by 2
+  };
```

Comment:
What about if rating is above 5?

Reply:
I think we should do this the other way.
Backend would have 0.0 - 5.0 and frontend 0 - 10. When loading on the frontend, just multiply by 2. When updating, divide by 2.